src/views/tickets/email.php

```diff
@@ -244,8 +266,13 @@
 					$break = 'page-break-before:always !important;';
 				}
 
-				$event      = get_post( $ticket['event_id'] );
-				$header_id  = get_post_meta( $ticket['event_id'], tribe( 'tickets.handler' )->key_image_header, true );
+				$event = get_post( $ticket['event_id'] );
+
+				/** @var Tribe__Tickets__Tickets_Handler $handler */
+				$handler = tribe( 'tickets.handler' );
+
+				$header_id  = get_post_meta( $ticket['event_id'], $handler->key_image_header, true );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 040dd73be46478fe02433e4de0c572d7</sub>